<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenGL</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/textures/ui/tank.png</normaloff>:/textures/ui/tank.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QOpenGLWidget" name="viewPort"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>496</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSystem">
    <property name="title">
     <string>System</string>
    </property>
    <addaction name="actionRendering"/>
    <addaction name="actionShow_console"/>
   </widget>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Unit test</string>
    </property>
    <addaction name="actionNew_game"/>
    <addaction name="actionTest_2"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionFast_game"/>
    <addaction name="actionFluid_terrain"/>
    <addaction name="actionLow_FPS"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuOptions"/>
   <addaction name="menuSystem"/>
  </widget>
  <action name="actionRendering">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Rendering</string>
   </property>
  </action>
  <action name="actionNew_game">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="actionFast_game">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fast game</string>
   </property>
  </action>
  <action name="actionShow_console">
   <property name="text">
    <string>Show console</string>
   </property>
  </action>
  <action name="actionFluid_terrain">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fluid terrain</string>
   </property>
  </action>
  <action name="actionLow_FPS">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Low FPS</string>
   </property>
  </action>
  <action name="actionTest_2">
   <property name="text">
    <string>2</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
